### PR TITLE
TiUP: fix grammar and format for topology.example.yaml

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

TiUP: fix grammar and format for topology.example.yaml

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/2846
- Other reference link(s):pingcap-incubator/tiup-cluster#298

### CODE REVIEW

File: how-to/deploy/orchestrated/tiup.md [hunk 1]
```diff
@@ -331,31 +331,31 @@ cat topology.yaml
 ```
 
 ```yaml
-# # Global variables are applied to all deployments and as the default value of
-# # them if the specific deployment value missing.
+# # Global variables are applied to all deployments and used as the default value of
+# # The deployments if a specific deployment value is missing.
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:335]:
```suggestion
# # the deployments if a specific deployment value is missing.
```
Please see my commits and comment in https://github.com/pingcap/docs-cn/pull/2846 and align them in this PR. Thanks!

Comment (toutdesuite) [reply how-to/deploy/orchestrated/tiup.md]: @yikeke The file is updated. PTAL

File: how-to/deploy/orchestrated/tiup.md [hunk 2]
```diff
@@ -457,7 +456,7 @@ tiflash_servers:
 #     data_dir: "/tidb-data/pump-8249"
 #     log_dir: "/tidb-deploy/pump-8249/log"
 #     numa_node: "0,1"
-#     # Config is used to overwrite the `server_configs.drainer` values
+#     # The following configs are used to overwrite the `server_configs.pd` values.
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:459]:
```suggestion
#     # The following configs are used to overwrite the `server_configs.drainer` values.
```

File: how-to/deploy/orchestrated/tiup.md [hunk 3]
```diff
@@ -466,13 +465,13 @@ tiflash_servers:
...
-#     # if drainer doesn't have checkpoint, use initial commitTS to initial checkpoint
-#     # will get a latest timestamp from pd if setting to be -1 (default -1)
+#     # If drainer doesn't have a checkpoint, use initial commitTS as the initial checkpoint.
+#     # Will get a latest timestamp from pd if commit_ts is set to -1 (the default value).
 #     commit_ts: -1
 #     deploy_dir: "/tidb-deploy/drainer-8249"
 #     log_dir: "/tidb-deploy/drainer-8249/log"
 #     numa_node: "0,1"
-#     # Config is used to overwrite the `server_configs.drainer` values
+#     # The following configs are used to overwrite the `server_configs.pd` values.
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:474]:
```suggestion
#     # The following configs are used to overwrite the `server_configs.drainer` values.
```

File: how-to/deploy/orchestrated/tiup.md [hunk 4]
```diff
@@ -849,34 +846,34 @@ pump_servers:
     port: 8250
     deploy_dir: "/tidb-deploy/pump-8249"
     data_dir: "/tidb-data/pump-8249"
-    # Config is used to overwrite the `server_configs.drainer` values
+    # The following configs are used to overwrite the `server_configs.pd` values.
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:849]: ditto

File: how-to/deploy/orchestrated/tiup.md [hunk 5]
```diff
@@ -849,34 +846,34 @@ pump_servers:
...
     config:
       gc: 7
   - host: 10.0.1.7
     ssh_port: 22
     port: 8250
     deploy_dir: "/tidb-deploy/pump-8249"
     data_dir: "/tidb-data/pump-8249"
-    # Config is used to overwrite the `server_configs.drainer` values
+    # The following configs are used to overwrite the `server_configs.pd` values.
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:857]: ditto

File: how-to/deploy/orchestrated/tiup.md [hunk 6]
```diff
@@ -849,34 +846,34 @@ pump_servers:
...
     config:
       gc: 7
   - host: 10.0.1.8
     ssh_port: 22
     port: 8250
     deploy_dir: "/tidb-deploy/pump-8249"
     data_dir: "/tidb-data/pump-8249"
-    # Config is used to overwrite the `server_configs.drainer` values
+    # The following configs are used to overwrite the `server_configs.pd` values.
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:865]: ditto

File: how-to/deploy/orchestrated/tiup.md [hunk 7]
```diff
@@ -849,34 +846,34 @@ pump_servers:
...
     port: 8249
     data_dir: "/tidb-data/drainer-8249"
-    # if drainer doesn't have checkpoint, use initial commitTS to initial checkpoint
-    # will get a latest timestamp from pd if setting to be -1 (default -1)
+    # If drainer doesn't have a checkpoint, use initial commitTS as the initial checkpoint.
+    # Will get a latest timestamp from pd if commit_ts is set to -1 (the default value).
     commit_ts: -1
     deploy_dir: "/tidb-deploy/drainer-8249"
-    # Config is used to overwrite the `server_configs.drainer` values
+    # The following configs are used to overwrite the `server_configs.pd` values.
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:876]: ditto

File: how-to/deploy/orchestrated/tiup.md [hunk 8]
```diff
@@ -1278,7 +1275,7 @@ tidb_servers:
     deploy_dir: "deploy/tidb-4000"
     log_dir: "deploy/tidb-4000/log"
     numa_node: "0,1"
-    # Config is used to overwrite the `server_configs.tidb` values
+    # The following configs are used to overwrite the `server_configs.pd` values.
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:1278]: ditto

File: quick-start-with-tidb.md [hunk 9]
```diff
@@ -189,9 +189,8 @@ Other requirements for the deployment machine:
     {{< copyable "shell-regular" >}}
 
     ```yaml
-    # # Global variables are applied to all deployments and as the default value of
-    # # them if the specific deployment value missing.
-     
+    # # Global variables are applied to all deployments and used as the default value of
+    # # the deployments if a specific deployment value is missing.
```
Comment (yikeke) [inline quick-start-with-tidb.md:193]: line 200 needs update.

**Pull request merged**